<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="Menu">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../../assets/icons/venda-rapida.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::NoTextInteraction</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Bahnschrift Light</family>
             <pointsize>15</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>Realizar Venda</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="nomeLabel">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Nome</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="InputName">
             <property name="maximumSize">
              <size>
               <width>550</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 1.5px solid rgb(26, 95, 180);
border-radius: 5px;
padding: 3px;
</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="preOLabel">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Preço</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="InputPrice">
             <property name="maximumSize">
              <size>
               <width>550</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 1.5px solid rgb(26, 95, 180);
border-radius: 5px;
padding: 3px;
</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="descriOLabel">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Observação</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="InputObservation">
             <property name="maximumSize">
              <size>
               <width>550</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 1.5px solid rgb(26, 95, 180);
border-radius: 5px;
padding: 3px;
</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="formaDePagamentoLabel">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Forma De Pagamento</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="InputPaymentMethod">
             <property name="font">
              <font>
               <family>Arial</family>
              </font>
             </property>
             <item>
              <property name="text">
               <string>Cartão De Crédito</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Cartão De Débito</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Dinheiro</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Pix</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="BtnAdd">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">.QPushButton{
	background-color: #FF512F;
	color: rgb(255, 255, 255);
	border: 5px solid#FF512F;
	border-radius: 15px;
	height: 25px;
	width: 95px;
}

.QPushButton:hover{
	background-color: #003973;
	border: 2px;
	color: white;
}

</string>
           </property>
           <property name="text">
            <string>Adicionar Venda</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="BtnEdit">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">.QPushButton{
	background-color: #FF512F;
	color: rgb(255, 255, 255);
	border: 5px solid#FF512F;
	border-radius: 15px;
	height: 25px;
	width: 75px;
}

.QPushButton:hover{
	background-color: #003973;
	border: 2px;
	color: white;
}

</string>
             </property>
             <property name="text">
              <string>Editar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BtnDelete">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">.QPushButton{
	background-color: #FF512F;
	color: rgb(255, 255, 255);
	border: 5px solid#FF512F;
	border-radius: 15px;
	height: 25px;
	width: 75px;
}

.QPushButton:hover{
	background-color: #003973;
	border: 2px;
	color: white;
}

</string>
             </property>
             <property name="text">
              <string>Apagar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="BtnGetDish">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">.QPushButton{
	background-color: #FF512F;
	color: rgb(255, 255, 255);
	border: 5px solid#FF512F;
	border-radius: 15px;
	height: 25px;
	width: 75px;
}

.QPushButton:hover{
	background-color: #003973;
	border: 2px;
	color: white;
}

</string>
           </property>
           <property name="text">
            <string>Pegar Dados</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="BtnClean">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">.QPushButton{
	background-color: #FF512F;
	color: rgb(255, 255, 255);
	border: 5px solid#FF512F;
	border-radius: 15px;
	height: 25px;
	width: 80px;
}

.QPushButton:hover{
	background-color: #003973;
	border: 2px;
	color: white;
}

</string>
           </property>
           <property name="text">
            <string>Limpar Dados</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="BtnGetMakeSale">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">.QPushButton{
	background-color: #FF512F;
	color: rgb(255, 255, 255);
	border: 5px solid#FF512F;
	border-radius: 15px;
	height: 25px;
	width: 75px;
}

.QPushButton:hover{
	background-color: #003973;
	border: 2px;
	color: white;
}

</string>
           </property>
           <property name="text">
            <string>Pegar Dados</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QTableWidget" name="DishTable">
           <property name="maximumSize">
            <size>
             <width>350</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="tabKeyNavigation">
            <bool>false</bool>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
           <property name="gridStyle">
            <enum>Qt::DashLine</enum>
           </property>
           <column>
            <property name="text">
             <string>Id</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Nome</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Preço</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="SalesTable">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="tabKeyNavigation">
            <bool>false</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
           <property name="gridStyle">
            <enum>Qt::DashLine</enum>
           </property>
           <column>
            <property name="text">
             <string>Id</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Nome</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Preço</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Observação</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Forma/Pagamento</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Data/Hora</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
